car-app-functional/src/test/resources/features/privateEndpointTest.feature

```diff
@@ -2,6 +2,8 @@ Feature: Testing private endpoint
 
   Scenario: the client calls "private/status"
```

Comment:
It would help if you could add a bit more information to this scenario - For example is this a happy case or a sad case?
```suggestion
  Scenario: the client calls "private/status" and receives a happy case response
```